f2/src/locales/en.json

```diff
@@ -181,6 +182,13 @@
   "howManyTimes.notSure": "I’m not sure",
   "howManyTimes.once": "Once",
   "howManyTimes.severalTimes": "Several times",
+  "ifAnonymousPage.intro": "This means we won't ask for your contact details and won't be able to follow up in the future.",
+  "ifAnonymousPage.title": "Report anonymously?",
+  "ifanonymous.nextButton": "Continue",
+  "ifanonymous.nextPage": "Next: How did the incident start?",
+  "ifanonymous.no": "No",
+  "ifanonymous.title": "Would you like to stay anonymous?",
+  "ifanonymous.yes": "Yes",
```

Comment:
Can we change all these new keys (here and in `fr.json`) to start with "anonymousPage", for example "anonymousPage.warning", "anonymousPage.intro", etc?

Reply:
Good idea!

Reply:
fixed